tidb-cloud/tidb-cloud-import-local-files.md

```diff
@@ -74,3 +74,21 @@ Currently, this method supports importing one CSV file for one task into either
 9. After the import task is completed, you can click **Explore your data by Chat2Query** to query your imported data. For more information about how to use Chat2Qury, see [Explore Your Data with AI-Powered Chat2Query](/tidb-cloud/explore-data-with-chat2query.md).
 
 10. On the **Import** page, you can click **View** in the **Action** column to check the import task detail.
+
+## FAQ
+
+### Can I only import some specified columns on TiDB Cloud?
+
+No. Currently, you can only import all columns of a CSV file into an existing table.
+
+In this case, you can use the MySQL client to connect your TiDB cluster, and then use [LOAD DATA](/sql-statements/sql-statement-load-data.md) to specify the columns to import. For example:
+
+```sql
+CREATE TABLE `import_test` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(64) NOT NULL,
+  `address` varchar(64) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
+LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE import_test FIELDS TERMINATED BY ',' (name, address);
```

Comment:
will report an error

```sql
mysql> CREATE TABLE `import_test` (
    ->   `id` int(11) NOT NULL AUTO_INCREMENT,
    ->   `name` varchar(64) NOT NULL,
    ->   `address` varchar(64) NOT NULL,
    ->   PRIMARY KEY (`id`)
    -> ) ENGINE=InnoDB;
Query OK, 0 rows affected (0.47 sec)

mysql> LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE import_test FIELDS TERMINATED BY ',' (name, address);
ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.
mysql>
```

Reply:
I am asking serverless team

Reply:
The problem is according to MySQL Client

> mysql  Ver 14.14 Distrib 5.7.40, for Linux (x86_64) using  EditLine wrapper
success
> mysql  Ver 8.0.26 for macos11 on arm64 (MySQL Community Server - GPL)
failed

If user encounter the failed case, they can add `--local-infile=true` when connect using `mysql`